### DISCUSSION TITLE
2022-10-06 ミーティング

### DISCUSSION THREAD

**Discussion by azu** [asciidwango/js-primer#1480] [Meeting]
2022-10-06のミーティングスレ

**Comment by azu** [comment]
前回からの差分

- [ fix(async): 非同期処理と同期処理の図を追加(並列) #1467 ](https://github.com/asciidwango/js-primer/pull/1467)
  - https://jsprimer.net/basic/async/
- [ fix(use-case/nodecli): ESModule対応 #1465 ](https://github.com/asciidwango/js-primer/pull/1465)
  - ES Module
- [ refactor: リポジトリをNode 16+/npm 8+へ変更 #1464 ](https://github.com/asciidwango/js-primer/pull/1464)
- [ fix: Atomの言及を削除 #1470 ](https://github.com/asciidwango/js-primer/pull/1470)

**Comment by azu** [comment]
やる必要があるもの

- [ Node.js 16 LTSに更新する #1437 ](https://github.com/asciidwango/js-primer/issues/1437)

**Comment by azu** [comment]
やれたらいいななもの

- [`letではなくconstで反復処理をする` のタイトルと内容の組み合わせは意図的なものでしょうか · Issue #1288 · asciidwango/js-primer](https://github.com/asciidwango/js-primer/issues/1288)
- [`function` キーワードと関数式のうち推奨のものがあればそれを、ない場合はその旨明記してほしい · Issue #1354 · asciidwango/js-primer](https://github.com/asciidwango/js-primer/issues/1354)
- [falsyな値の一覧の不足 · Issue #1378 · asciidwango/js-primer](https://github.com/asciidwango/js-primer/issues/1378)
- [ECMAScript: ECMA262のURL変更 · Issue #1479 · asciidwango/js-primer](https://github.com/asciidwango/js-primer/issues/1479)
- [Array#sliceの説明について · Issue #1163 · asciidwango/js-primer](https://github.com/asciidwango/js-primer/issues/1163)

**Reply by azu** [reply]
次の値は、falseとなることが決まっています。

...

これらの値のことをfalsyな値と呼びます。

**Reply by lacolaco** [reply]
falsyな値についての記述を少し改訂トライしてみる

**Comment by azu** [comment]
将来ありそうなこと: Intl の対応とか

**Comment by azu** [comment]
付録: 将来入ってそうなProposalについて
とか
https://github.com/asciidwango/js-primer/issues/957
